src/main/webapp/main/AuthenticatedRoute.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+import { Storage } from '../util/Storage';
+
+const AuthenticatedRoute = ({ component: Component, props: componentProps, admin, ...rest }) => {
+
+  const { path, location } = rest;
+
+  return (
+    <Route
+      path={path}
+      location={location}
+      render={
+        props =>
+       verifyUser(admin, props)
```

Comment:
nit: spacing here, and also throughout the file. In intelli-j, `command + option + L` will reformat the code with the correct eslint settings defined for the project.